BooruSharp/Booru/Template/Gelbooru.cs

```diff
@@ -15,8 +15,12 @@ namespace BooruSharp.Booru.Template
     /// </summary>
     public abstract class Gelbooru : ABooru
     {
-        public Gelbooru(string url, params BooruOptions[] options) : base(url, UrlFormat.indexPhp, CombineArrays(options,
-            new[] { BooruOptions.noWiki, BooruOptions.noRelated, BooruOptions.limitOf20000, BooruOptions.commentApiXml }))
+        [Obsolete(_deprecationMessage)]
+        public Gelbooru(string url, params BooruOptions[] options) : this(url, MergeOptions(options))
```

Comment:
Maybe the params keyword should be remove here? (and in all the others template classes)

Because it's a bit confusing if you do Gelbooru("someUrl"), you're not really sure which ctor will be called

Reply:
In case only one `BooruOptions` object is supplied (or isn't supplied at all), the `ctor(string url, BooruOptions options)` will take precedence over `ctor(string url, params BooruOptions[] options)`.

The idea I had in mind is to keep this fully backwards compatible. Say, someone could create their own booru and pass parameters to the base class in the constructor like so: `base("my.api.url", BooruOptions.noWiki, BooruOptions.useHttp)`, that will use `ctor(string url, params BooruOptions[] options)` overload. If we were to remove `params` from these constructors one would have to pass `base("my.api.url", new[] { BooruOptions.noWiki, BooruOptions.useHttp })` instead (which isn't desirable anymore with this PR), and at that point I think we can just remove these overloads completely and leave just the `ctor(string url, BooruOptions options)` variants.

What are your thoughts?

Reply:
Yeah you're right.

The rest of the code looks great so I'll merge the PR and publish a new version under BooruSharp 2.9.0.

Thanks a lot for your contribution!